wordpress-mobile/WordPress-iOS
# update full screen (No results view) copy in offline mode

Fixes #11030 
We want to show the same copy when the full screen (no results view) is displayed in case of connectivity issue.
To test:
1. Run the app.
2. Go over any of the full pages and see that the copy is the same.
3. Check that the full screen copy is the same in case the screen appears due to other reason (will update testing steps)

* This PR does not include the updated illustration
Update release notes:
